app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -296,10 +295,8 @@ private fun WallpaperThumbnailsPreview() {
         val wallpaperManager = context.components.wallpaperManager
```

Comment:
the preview isn't building. let me know if you wanna pair up on this.

Reply:
I'll just need to remember to check them in the future. I ran into problems with my M1 when I got started and got in the habit of not doing so lol. I'll take a look and let you know if I could use some help, thanks 🙂

Reply:
This should be working now! We are planning to make all wallpapers sourced remotely, so unfortunately there isn't much to show. But it at least renders!